src/sbt-test/sbt-reactive-app/akka-cluster-endpoints/build.sbt

```diff
@@ -12,7 +12,8 @@ lazy val root = (project in file("."))
     ),
 
     packageName in Docker := "hello-akka",
-    enableAkkaClusterBootstrap := Some(true)
+    enableAkkaClusterBootstrap := Some(true),
+    akkaClusterBootstrapSystemName := Some("hey")
```

Comment:
This adds a new Dockerfile label that should be verified in this test:

`LABEL com.lightbend.rp.akka-cluster-bootstrap.system-name="hey"`

Reply:
:blush: that was the intent, looks like I forgot to add it. Thanks for the catch!